core/src/dird/dird.cc

```diff
@@ -395,7 +395,17 @@ int main(int argc, char* argv[])
     goto bail_out;
   }
 
-  if (test_config) { TerminateDird(0); }
+  if (test_config) {
+    if (my_config->HaveDeprecationWarnings()) {
+      // FIXME: cannot use Jmsg here as only M_FATAL and M_ERROR_TERM will
+      //        work because daemon_msgs is not initialized
+      fprintf(stderr, _("Deprecated configuration settings detected:\n"));
```

Comment:
When running as a daemon stderr could have been closed.

Reply:
`if (test_config)` gurarantees that we haven't daemonized, because `daemon_start()` is called in a `if (!test_config)`-block. And even if `daemon_start()` had already run stderr would be still open and point to `/dev/null` (see `SetupStdFileDescriptors()` in `core/src/lib/daemon.cc`.